cfme/utils/appliance/implementations/ssui.py

```diff
@@ -127,10 +133,16 @@ def go(self, _tries=0):
             self.prerequisite_view = self.prerequisite()
             logger.debug("SSUI-NAVIGATE: Heading to destination {}".format(self._name))
             self.do_nav(_tries)
-        self.resetter()
+        if nav_args['use_resetter']:
+            self.resetter()
         self.post_navigate(_tries)
-        if self.VIEW is not None:
```

Comment:
So if this is false, the method returns None anyway, so might as well return self.view if its None too.